setup.py

```diff
@@ -1,43 +1,43 @@
 #!/usr/bin/env python3
 
 try:
-	from setuptools import setup
+    from setuptools import setup
 except ImportError:
-	from distutils.core import setup
+    from distutils.core import setup
 
 import os
 import sys
+
 import libgrabsite
 
 install_requires = [
-	"click>=6.3",
-	"wpull @ https://github.com/ArchiveTeam/ludios_wpull/archive/refs/tags/3.0.9.zip",
-	"manhole>=1.0.0",
-	"lmdb>=0.89",
-	"autobahn>=0.12.1",
-	"google-re2>=1.0.6",
-	"websockets>=6.0",
+    "click>=8.0.0",
+    "wpull @ https://github.com/ngie-eign/ludios_wpull/archive/py313-support.zip",
```

Comment:
Would you consider switching that to e.g.

```suggestion
    "wpull~=5.0.4",
```

?

The MR https://github.com/ArchiveTeam/ludios_wpull/pull/36 is merged + released now.

And it seems that this

```py
    "wpull @ https://github.com/ngie-eign/ludios_wpull/archive/py313-support.zip",
```

doesn't work anymore

```console
$ uvx --python=3.8 --no-binary-package lxml git+https://github.com/ngie-eign/grab-site@py313-support
    Updated https://github.com/ngie-eign/grab-site (e81eed8c37616492466848ab5ed68104c7339693)
  × Failed to download and build `wpull @ https://github.com/ngie-eign/ludios_wpull/archive/py313-support.zip`
  ├─▶ Failed to fetch: `https://github.com/ngie-eign/ludios_wpull/archive/py313-support.zip`
  ╰─▶ HTTP status client error (404 Not Found) for url (https://codeload.github.com/ngie-eign/ludios_wpull/zip/py313-support)
```

Reply:
Actually, `wpull~=5.0.4` does not exist in PyPI (https://pypi.org/project/wpull/ only a very old version of it).

I guess the correct solution would be

```
git+https://github.com/ArchiveTeam/ludios_wpull@5.0.4
```